examples/migraphx/migraphx_driver/README.md

```diff
@@ -55,7 +55,9 @@ See below for a comprehensive list of commands and option arguments, as well as
 | --exhaustive-tune                        | Enable exhaustive search to find fastest kernel           |
 | --fp16                                   | Quantize for fp16                                         |
 | --int8                                   | Quantize for int8                                         |
-| --tolerance                              | Tolerance for errors                                      |
+| --rms_tol                                | Tolerance for the RMS error                               |
```

Comment:
Are these three new options exclusive? Thanks.

Reply:
Exclusive you mean if they all are different settings ? then yes.